### DISCUSSION TITLE
Welcome to cinema-app Discussions!

### DISCUSSION THREAD

**Discussion by alianu23** [Leap-MERN-2023/cinema-app#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by amarbold22** [comment]
sain uu

**Reply by alianu23** [reply]
sn snu

**Reply by Bilguungantogos** [reply]
gal2

**Reply by amarbold22** [reply]
issue uusgesen. transition deer ajillaa

**Reply by alianu23** [reply]
{
  "workbench.colorTheme": "Default Dark+",
  "editor.minimap.autohide": true,
  "editor.accessibilitySupport": "off",
  "files.autoSave": "onFocusChange",
  "editor.formatOnSave": true,
  "editor.defaultFormatter": "esbenp.prettier-vscode",
  "workbench.iconTheme": "material-icon-theme",
  "javascript.updateImportsOnFileMove.enabled": "always",
  "git.autofetch": true,
  "tailwindCSS.emmetCompletions": true,
  "editor.inlineSuggest.enabled": true,
  "editor.quickSuggestions": {
    "strings": true
  },
  "css.validate": false,
  "[dart]": {
    "editor.formatOnSave": true,
    "editor.formatOnType": true,
    "editor.rulers": [80],
    "editor.selectionHighlight": false,
    "editor.suggestSelection": "first",
    "editor.tabCompletion": "onlySnippets",
    "editor.wordBasedSuggestions": "off"
  },
  "explorer.confirmDelete": false
}
